src/scores/continuous/standard_impl.py

```diff
@@ -354,11 +354,10 @@ def pbias(
     """
     Calculates the percent bias, which is the ratio of the additive bias to the mean observed value, multiplied by 100.
 
-    Percent bias is used for evaluating and comparing forecast accuracy across stations or dataset with varying magnitudes.
-    By expressing the error as a percentage of the observed value, it allows for standardized comparisons, enabling assessment
-    of forecast performance regardless of the absolute scale of values. Like multiplicative_bias, Percent bias will return a np.inf
-    where the mean of `obs` across the dims to be reduced is 0.
-    It is defined as
+    Percent bias is used for evaluating and comparing forecast accuracy across stations or datasets with varying magnitudes.
+    By expressing the error as a percentage of the observed value, it allows for standardised comparisons, enabling assessment
+    of forecast performance regardless of the absolute scale of values. Like :py:func:`scores.continuous.multiplicative_bias`,
+    ``pbias`` will return a np.inf where the mean of ``obs`` across the dims to be reduced is 0. It is defined as
 
     .. math::
         \\text{Percent bias} = 100 * \\frac{\\sum_{i=1}^{N}(x_i - y_i)}{\\sum_{i=1}^{N} y_i}
```

Comment:
Should the "*" be replaced by another symbol, such as \\\times or \\\cdot? I don't have an opinion on this - I just wanted to raise the question - so will leave you, @tennlee and @nicholasloveday make a decision on that.